### ISSUE TITLE
bazel test failed every other run

### ISSUE THREAD

**Issue by ddwolf** [bazelbuild/bazel#2166]
I was running bazel test, it run into error every other time, here is the output, do you guys know what's the problem?
[part1](https://github.com/ddwolf/pastebin/wiki)
[part2](https://github.com/ddwolf/pastebin/wiki/part2)
* command: *
```
export ENCRYPT_KEY=password
bazel --max_idle_secs=0 test :alltests \
--spawn_strategy=standalone --test_strategy=standalone --test_env ENCRYPT_KEY
```

### Environment info
* Operating System:
UBUNTU14.04

* Bazel version (output of `bazel info release`):
release 0.4.0- (@non-git)

I downloaded from github with tag 0.4.0

**Comment by iirina** [comment]
Hi,

Did you run the same command twice and the second time you got an error? If not, can you please specify both commands that you ran and if you did something else in between (setting some env variables, ran other commands, etc).

Thanks,
Irina

**Comment by ddwolf** [comment]
@iirina I didn't do any thing between the two times run. *I run the same command twice and the second time I got an error.*

**Comment by iirina** [comment]
One more question: did you edit any BUILD file in the project in between? 
I did a little research and this seems to be a known issue of `bazel 0.4.0` that was fixed in `0.4.1`. Could you try this with `0.4.1 `and see if the problem persists? Thanks!

**Comment by pcj** [comment]
fyi I've seen this before, bazel would flip/flop between success/fail each time I ran a command. Happened a long time ago, can't remember how to reproduce.

**Comment by ddwolf** [comment]
@iirina: I didn't edit any BUILD file in between.and thanks for the reply, I 'll try 0.4.1.

**Comment by iirina** [comment]
OK. Please let me know if you still have problems.

**Comment by ddwolf** [comment]
@iirina: the error was fixed in 0.4.1, thanks a lot.
